test/controllers/sessions_controller_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'test_helper'
+
+class SessionsControllerTest < ActionController::TestCase
+  def setup
+    request.env['omniauth.auth'] = OmniAuth.config.mock_auth[:github]
+  end
+
+  test 'new should redirect to /auth/github' do
```

Comment:
Kind of a personal preference of mine not to use `should` in test case descriptions as it's unnecessary. `#new redirects to /auth/github`